### PR TITLE
Set up application structure for public client

### PR DESCRIPTION
**Pull request by 1CIM**
PT: https://www.pivotaltracker.com/story/show/176030772

## User story
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
```
Comment (emtalen) [inline public/index.html:19]: Clean the comments

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const App = () => {
+  return <div>
```
Comment (emtalen) [inline src/App.jsx]: Format this code

**Review by emtalen — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (1CIM) [reply public/index.html:19]: done

File: src/App.jsx (hunk 2 above)
Comment (1CIM) [reply src/App.jsx]: done

**Review by emtalen — APPROVED** [review]

**Pull request merged**